### PR TITLE
User interface: "Advanced" menu tooltips

### PR DESCRIPTION
**Pull request by PilotFlying**
Added tooltips to the "Remove" and "Uninstall" options in the advanced menu. As a new user, I had to revisit the [documentation](https://docs.abapgit.org/guide-online-uninstall.html) to distinguish between the two.

The tooltips are passed as "iv_title" parameters to ZCL_ABAPGIT_HTML_TOOLBAR->add, I am not sure if the parameters are intended for this (but they did produce tooltips)

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap [hunk 1]
```diff
@@ -309,6 +309,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
     ENDIF.
 
     ro_advanced_dropdown->add( iv_txt = 'Remove'
+                               iv_title = `Remove abapGit's records of the repository (SAP objects unaffected)`
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:312]:
```suggestion
                               iv_title = `Remove abapGit's records of the repository (system objects will be unaffected)`
```

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap [hunk 2]
```diff
@@ -317,6 +318,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
       lv_crossout = zif_abapgit_html=>c_html_opt-crossout.
     ENDIF.
     ro_advanced_dropdown->add( iv_txt = 'Uninstall'
+                               iv_title = `Deletes SAP package objects!`
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]:
```suggestion
                               iv_title = `Delete all objects in this package from the system`
```

**Review by FreHu — COMMENTED** [review]
I'd suggest not to call it 'SAP objects' but 'system objects'

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 2 above)
Comment (mbtools) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: ... and of all subpackages

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 1 above)
Comment (flaiker) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:312]: I think "development object(s)" is the term used in other environments (SE80/ADT).

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 2 above)
Comment (flaiker) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: ... depending on the ignore subpackages setting

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 1 above)
Comment (PilotFlying) [inline src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:312]: Thank you! How is this?
```suggestion
                               iv_title = `Remove abapGit's records of the repository (the system's development objects will remain unaffected)`
```

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 2 above)
Comment (PilotFlying) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: In line with the above, what do you think of:
```suggestion
                               iv_title = `Delete all development objects belong to this package (and subpackages) from the system!`
```

Both this and the "remove" suggestion above, seem to fit:
image

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 1 above)
Comment (PilotFlying) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:312]: Thank you! How is this?
```suggestion
                               iv_title = `Remove abapGit's records of the repository (the system's development objects will remain unaffected)`
```
image

File: src/ui/zcl_abapgit_gui_page_repo_view.clas.abap (hunk 2 above)
Comment (FreHu) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: i don't like exclamation marks in documentation, but maybe it's just a personal preference

Comment (mbtools) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: +1 to avoid exclamation marks!

Comment (PilotFlying) [reply src/ui/zcl_abapgit_gui_page_repo_view.clas.abap:321]: Copy that, I don't like them either frankly. I thought an exception was warranted, to caution people as uninstall is arguably "dangerous". But I suppose the confirmation dialog which follows covers the concern. 
So:
```suggestion
                               iv_title = `Delete all development objects belonging to this package (and subpackages) from the system`
```

**Review by mbtools — APPROVED** [review]
LGTM

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**